Tests/installation_tests/cocoapods/with_frameworks_swift/CocoapodsTest/Info.plist

```diff
@@ -22,6 +22,8 @@
 	<string>1</string>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
+	<key>NSCameraUsageDescription</key>
+	<string>camera description</string>
```

Comment:
Connect assertion fails if the app doesn't configure this, so this has been added to all installation test projects